src/bin/tikv-ctl.rs

```diff
@@ -174,9 +174,26 @@ trait DebugExecutor {
         println!("entry {:?}", entry);
         println!("msg len: {}", data.len());
 
-        let mut msg = RaftCmdRequest::new();
-        msg.merge_from_bytes(&data).unwrap();
-        println!("{:?}", msg);
+        if data.is_empty() {
+            return;
+        }
+
+        match entry.get_entry_type() {
+            EntryType::EntryNormal => {
+                let mut msg = RaftCmdRequest::new();
+                msg.merge_from_bytes(&data).unwrap();
+                println!("{:?}", msg);
+            }
+            EntryType::EntryConfChange => {
+                let mut msg = ConfChange::new();
+                msg.merge_from_bytes(&data).unwrap();
+                let ctx = msg.take_context();
+                println!("{:?}", msg);
```

Comment:
`println!("ConfChange: {:?}", msg)` is better?

Reply:
make sense, besides i suggest to use pretty print ":#?", but it can't control the ident level.

it will like this: 

nomal entry
```
idx_key: \001\002\000\000\000\000\000\000\000\t\001\000\000\000\000\000\000\000\010
region: 9
log index: 8
entry term: 6 index: 8
msg len: 270
Normal: header {
  region_id: 9
  peer {
    id: 10
    store_id: 1
  }
  region_epoch {
    conf_ver: 1
    version: 3
  }
  term: 6
}
requests {
  cmd_type: Delete
  delete {
    cf: "lock"
    key: "t\200\000\000\000\000\000\000\377\005_r\200\000\000\000\000\377\000\000\001\000\000\000\000\000\372"
  }
}
requests {
  cmd_type: Put
  put {
    cf: "write"
    key: "t\200\000\000\000\000\000\000\377\005_r\200\000\000\000\000\377\000\000\001\000\000\000\000\000\372\372}L\345\347?\377\376"
    value: "P\205\200\300\271\241\343\254\301\005"
  }
}
requests {
  cmd_type: Delete
  delete {
    cf: "lock"
    key: "t\200\000\000\000\000\000\000\377\005_i\200\000\000\000\000\377\000\000\001\001%\000\000\000\377\000\000\000\000\370\001ro\377ot\000\000\000\000\373\000\376"
  }
}
requests {
  cmd_type: Put
  put {
    cf: "write"
    key: "t\200\000\000\000\000\000\000\377\005_i\200\000\000\000\000\377\000\000\001\001%\000\000\000\377\000\000\000\000\370\001ro\377ot\000\000\000\000\373\000\376\372}L\345\347?\377\376"
    value: "P\205\200\300\271\241\343\254\301\005v\010\000\000\000\000\000\000\000\001"
  }
}
```

confchange type
```
idx_key: \001\002\000\000\000\000\000\000\000\t\001\000\000\000\000\000\000\000\t
region: 9
log index: 9
entry entry_type: EntryConfChange term: 6 index: 9 sync_log: true
msg len: 32
ConfChange: node_id: 35
ConfChange.RaftCmdRequest: header {
  region_id: 9
  peer {
    id: 10
    store_id: 1
  }
  region_epoch {
    conf_ver: 1
    version: 3
  }
}
admin_request {
  cmd_type: ChangePeer
  change_peer {
    peer {
      id: 35
      store_id: 5
    }
  }
}
```